src/test/stemcontrolobjecttest.cpp

```diff
@@ -0,0 +1,236 @@
+#include <gtest/gtest.h>
+
+#include <QScopedPointer>
+#include <QtDebug>
+#include <memory>
+
+#include "control/pollingcontrolproxy.h"
+#include "mixxxtest.h"
+#include "test/signalpathtest.h"
+
+class StemControlTest : public BaseSignalPathTest {
+  protected:
+    void SetUp() override {
+        BaseSignalPathTest::SetUp();
+
+        const QString kStemFileLocationTest = getTestDir().filePath("stems/test.stem.mp4");
+        TrackPointer pStemFile(Track::newTemporary(kStemFileLocationTest));
+
+        loadTrack(m_pMixerDeck1, pStemFile);
+        loadTrack(m_pMixerDeck3, pStemFile);
+
+        m_pPlay = std::make_unique<PollingControlProxy>(m_sGroup1, "play");
+        m_pStem1Volume = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_1_volume");
+        m_pStem2Volume = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_2_volume");
+        m_pStem3Volume = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_3_volume");
+        m_pStem4Volume = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_4_volume");
+        m_pStem1Mute = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_1_mute");
+        m_pStem2Mute = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_2_mute");
+        m_pStem3Mute = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_3_mute");
+        m_pStem4Mute = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_4_mute");
+        m_pStem1Color = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_1_color");
+        m_pStem2Color = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_2_color");
+        m_pStem3Color = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_3_color");
+        m_pStem4Color = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_4_color");
+        m_pStemCount = std::make_unique<PollingControlProxy>(m_sGroup1, "stem_count");
```

Comment:
Before I merge this PR, which introduces new COs, I would like to hear your opinion about the naming scheme of the stem COs ( the full list can be found in the description of https://github.com/mixxxdj/mixxx/issues/13116 ).
Do you prefer the style:
`[Channel1],stem_1_volume` or `[Channel1Stem1],volume`
The first has the benefit, that the group name is similar to other COs in the mappings,
the second would have the benefit, that we seperate channel and function clearer and we could add it as just another group syntax to the documentation of some existing COs.
The first syntax would be in line with `[Channel1],hotcue_X_color`, the second with `[QuickEffectRack1_[ChannelXStemY]],super1`.
If we go with the first syntax, than I think `[QuickEffectRack1_[Channe1Stem1]],super1` should become `[QuickEffectRack1_[Channel1]],stem1_super1` for consistency.
What's your opinion?

Reply:
Honestly, I think using `[ChannelXStemY]` globally would make a lot of sense, but I remember stumbling across something and settling for `[ChannelX],stem_Y_...`. 
Now that my Mixxx knowledge has grew, I will retry and see if I can get `ChannelXStemY` working for consistency.

Reply:
Ok, that was surprisingly easy finally, I do not know how come I didn't do this before :sweat_smile: 
Also updated [the manual](https://github.com/mixxxdj/manual/pull/644) and will update the epic right away.